perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -594,6 +594,14 @@ sub waste_container_actions {
     };
 }
 
+sub waste_recently_expired {
+    my ($self, $schedule) = @_;
+
+    my ($y, $m, $d) = ( $schedule->{end_date} =~ /(\d+)-(\d+)-(\d+)/);
+    my $end = DateTime->new( year => $y, month => $m, day => $d)->set_time_zone(FixMyStreet->local_time_zone);
+    return $self->waste_sub_overdue($end, weeks => 4);
```

Comment:
waste_sub_overdue is given a string, not a DateTime, so can you drop waste_recently_expired entirely, and only call `waste_sub_overdue($schedules->{end_date}, weeks => 4)` or have I missed something?

Reply:
for some reason I was convinced that DateTime::Format::W3CDTF didn't parse that date but it turns out I was wrong :|